src/shared/user-agent.js

```diff
@@ -10,3 +10,28 @@
 export const isMacOS = (_userAgent = window.navigator.userAgent) => {
   return _userAgent.indexOf('Mac OS') >= 0;
 };
+
+/**
+ * Returns true when device is iOS.
+ * https://stackoverflow.com/a/9039885/14463679
```

Comment:
I worry that this may be a little brittle or get out of date in future. If we could think of a way to do this without specific iDevice detection that would be preferable, but since this is a very popular StackOverflow answer at least there will be many other websites using the same approach.